protos/node_reputation/node_reputation.proto

```diff
@@ -0,0 +1,85 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+package nodereputation;
+
+service NodeReputation {
+    rpc NodeReputation(NodeQuery) returns (NodeReputationRecord) {}
+    rpc UpdateReputation(NodeUpdate) returns (UpdateReply) {}
+    rpc FilterNodeReputation(NodeFilter) returns (NodeReputationRecords) {}
+    rpc PruneNodeReputation(NodeQuery) returns (UpdateReply) {}
+}
+
+message NodeReputationRecord {
+    string source = 1;
+    string nodeName = 2;
+	string timestamp = 3;
+	int64 uptime = 4;
+	int64 auditSuccess = 5;
+	int64 auditFail = 6;
+	int64 latency = 7;
+	int64 amountOfDataStored = 8;
+	int64 falseClaims = 9;
+    int64 shardsModified = 10;
+    float score = 11;
+}
+
+message NodeReputationRecords {
+    repeated NodeReputationRecord records = 1;
+}
+
+message UpdateReply {
+    string bridgeName = 1;
+    string nodeName = 2;
+    string source = 3;
+
+    enum ReplyType {
+        UPDATE_SUCCESS = 0;
+        UPDATE_FAILED = 1;
+    }
+
+    ReplyType status = 4;
```

Comment:
you don't need to handle this yourself. grpc has a status and a code system for returning errors separately from your response protobuf